docs/modules/servers/pages/local/conf/index.adoc

```diff
@@ -1,23 +1,99 @@
+
+
+
+Scenarios for address rewriting
+
+The following scenarios are examples of how you can use address rewriting:
+
+Group consolidation: Some organizations segment their internal businesses into separate domains that are based on business or technical requirements. This configuration can cause email messages to appear as if they come from separate groups or even separate organizations.
+
+The following example shows how an organization, Contoso, Ltd., can hide its internal subdomains from external recipients:
+
+Outbound messages from the northamerica.contoso.com, europe.contoso.com, and asia.contoso.com domains are rewritten so they appear to originate from a single contoso.com domain. All messages are rewritten as they pass through Edge Transport servers that provide SMTP connectivity between the whole organization and the Internet.
+
+Inbound messages to contoso.com recipients are relayed by the Edge Transport server to a Mailbox server. The message is delivered to the correct recipient based on the proxy address that's configured on the recipient's mailbox.
+
+Mergers and acquisitions: An acquired company might continue to run as a separate business, but you can use address rewriting to make the two organizations appear as if they're one integrated organization.
+
+The following example shows how Contoso, Ltd. can hide the email domain of the newly acquired company, Fourth Coffee:
+
+Contoso, Ltd. wants all outbound messages from Fourth Coffee's Exchange organization to appear as if they originate from contoso.com. All messages from both organizations are sent through the Edge Transport servers at Contoso, Ltd., where email messages are rewritten from user@fourthcoffee.com to user@contoso.com.
+
+Inbound messages to user@contoso.com are rewritten and routed to user@fourthcoffee.com mailboxes. Inbound messages that are sent to user@fourthcoffee.com are routed directly to Fourth Coffee's email servers.
+
+Partners: Many organizations use external partners to provide services for their customers, other organizations, or their own organization. To avoid confusion, the organization might replace the email domain of the partner organization with its own email domain.
+
+The following example shows how Contoso, Ltd. can hide a partner's email domain:
+
+Contoso, Ltd. provides support for the larger Wingtip Toys organization. Wingtip Toys wants a unified email experience for its customers, and it requires all messages from support personnel at Contoso, Ltd. to appear as if they were sent from Wingtip Toys. All outbound messages that relate to Wingtip Toys are sent through their Edge Transport servers, and all contoso.com email addresses are rewritten to wingtiptoys.com email addresses.
+
+Inbound messages for support@wingtiptoys.com are accepted by Wingtip Toy's Edge Transport servers, rewritten, and then routed to the support@contoso.com email address.
```

Comment:
You nicely developed real word usage for domain aliases :-)

Maybe we need to specifically detail that these examples are for Domain Aliases, and we might want to detail other Recipient Rewritting options as well.

Reply:
Hahahaha. Actually I copy-pasted this from some website and forgot to update the text. I thought it was interesting and useful information. Either we should link to that source or write our own text, else it would be plagiarism. :-)

Reply:
(plagiarism is only an issue if there is some licensing issues ;-) )

Well giving some insight about real world situation and associate it to our *recipient rewrite table* wording is a must have.

Reply:
Yes, for sure. I am planning on updating this text and, as you pointed out, adding a section to "Concepts".

This is also the reason why I want to have a minimalistic Basic server. If everything needs to be backed by a "Concepts" text, that is a lot of work! :-)